person.rb

```diff
@@ -1,25 +1,34 @@
 require './nameable'
+require './book'
 require './capitalizer'
 require './trimdecorator'
 require './decoratorbase'
+require './rental'
+
 # Make sure that your Person class inherits from Nameable
 
 class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
+
   def initialize(id, name = 'Unknown', age = 0, parent_permission: true)
     super()
     @id = id
     @name = name
     @parent_permission = parent_permission
     @age = age
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    rentals << rental
+    rental.book.add_rental(rental)
   end
```

Comment:
Please can you use this method for adding rentals?
```ruby
 def add_rental(book, date)
    Rental.new(date, book, self)  #The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
  end
```
[See examples through this link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)